libs/npm/core/src/hooks/keycloak/useKeycloakWrapper.ts

```diff
@@ -13,11 +13,12 @@ export function useKeycloakWrapper(): IKeycloak {
 
   const [initialized, setInitialized] = React.useState(ready);
 
+  if (ready) token = keycloak.tokenParsed as IKeycloakToken;
```

Comment:
Fix state issue with token value.  The token is stored as a global module level so that state doesn't get refreshed all the time for each instance of the hook.